packages/terra-toggle-button/src/ToggleButton.module.scss

```diff
@@ -5,10 +5,19 @@
 
   .icon {
     display: inline-block;
-    font-size: 0.8em;
+    line-height: 1rem;
     transform: rotate(0deg);
-  }
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+      line-height: 1.35rem;
+    }
 
+    svg {
+      font-size: 0.8em;
+      position: relative;
+      top: -0.15em;
+    }
+  }
```

Comment:
Media-query applied in the wrong spot, updated values:
```
    svg {
      font-size: 0.8em;
      position: relative;
      top: -0.07143em;
      @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
          top: -0.14285em;
      }
    }
  }
```

Reply:
Resolved in 5ebdfe6